grakn-engine/src/main/java/ai/grakn/engine/controller/VisualiserController.java

```diff
@@ -109,7 +110,7 @@ public VisualiserController(Service spark) {
     private String conceptById(Request req, Response res) {
         String keyspace = getKeyspace(req);
 
-        try (GraknGraph graph = getInstance().getGraph(keyspace)) {
+        try (GraknGraph graph = getInstance().getGraph(keyspace, GraknTxType.WRITE)) {
```

Comment:
This one (and others) can probably be read-only.  This may be something for another PR though.